backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -93,23 +95,36 @@ def resolve_fetch_failures_0_members(
         # Fetch data
         csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(
             codeset_ids=list(failed_cset_ids), flag_issues=False)
-        # - filter success & track results
-        csets_and_members['OMOPConceptSet'] = [
-            x for x in csets_and_members['OMOPConceptSet'] if x['member_items']]
-        cset_is_draft_map.update({x['properties']['codesetId']: x['properties']['isDraft'] for x in csets_and_members['OMOPConceptSet']})
-        success_cases: List[int] = [
-            x['properties']['codesetId'] for x in csets_and_members['OMOPConceptSet']]
-        failed_cset_ids = list(set(failed_cset_ids) - set(success_cases))
+
+        # - identify discarded drafts
+        # TODO: ascertain discarded drafts
+        #  - how to ascertain?
+        #    - 1 (ok~): detect discarded drafts as not returned from fetch_cset_and_member_objects(). ~confirm w/ amin et al
+        #    - 2 (x): Ask Amin et al: is the enclave API reliable for discarded drafts? Some of these are appearing in the
+        #    enclave API but not in the UI. x: I think (2) is not a case I need to worry about. i think they weren't discared.
+        #    - This line below should be enough to handle. Just want to verify (1)
+        discarded_cset_ids: Set[int] = set([x for x in failed_cset_ids if x not in csets_and_members['OMOPConceptSet']])
+        # TODO: mark discarded drafts resolved
```

Comment:
## Question: Run `refresh --since` longer than Jan 1?
I got 20 new fetch failures because, after fixing a bug in which drafts were being erroneously marked as resolved (we would not fetch their members), I ran this since the last time I remember running a big refresh (some time in January). However, the last time we initialized was last June or July I think. Maybe I need to do `refresh --since` back to that time, just in case? I suppose so.

Reply:
I decided to do this. The last time we initialized was `2023-07-12T03:51:48.292700-04:00`, so I decided to run this since `2023-07-11`. I also decided it was time to be able to run this via GitHub actions, so I created:
- #760

Reply:
I ran it, but I was surprised to see [very few deltas](https://github.com/jhu-bids/TermHub/blob/develop/docs/backend/db/analysis.md).

It looks like it was able to fetch many long-lived drafts (which we had previously erroneously marked finalized): 37 new ones in this run.